File: apps/client/webui/src/types/models/host.ts
export type Host = {
  id: string;
  ip_address: string;
  port: number;
  name: string;
  status: "OFFLINE" | "AVAILABLE" | "INUSE" | "DISABLED";
};

export type SimpleHost = Omit<Host, "ip_address" | "port">;

File: apps/client/internal/models/host.go
package models

type Host struct {
	ID        string `json:"id"`
	IPAddress string `json:"ip_address"`
	Name      string `json:"name"`
	Port      int    `json:"port"`
	Status    string `json:"status"`
}

type SimpleHost struct {
	ID     string `json:"id"`
	Name   string `json:"name"`
	Status string `json:"status"`
}
